src/VRChat.API/Model/APIConfigConstantsGROUPS.cs

```diff
@@ -44,7 +45,7 @@ public partial class APIConfigConstantsGROUPS : IEquatable<APIConfigConstantsGRO
         /// <param name="mAXMANAGEMENTROLES">Maximum number of management roles in a group (default to 5).</param>
         /// <param name="mAXOWNED">Maximum number of groups a user can own (default to 5).</param>
         /// <param name="mAXROLES">Maximum number of roles in a group (default to 50).</param>
-        public APIConfigConstantsGROUPS(int cAPACITY = 100000, List<string> gROUPTRANSFERREQUIREMENTS = default(List<string>), int mAXINVITESREQUESTS = 50, int mAXJOINED = 100, int mAXJOINEDPLUS = 200, int mAXLANGUAGES = 10, int mAXLINKS = 3, int mAXMANAGEMENTROLES = 5, int mAXOWNED = 5, int mAXROLES = 50)
+        public APIConfigConstantsGROUPS(int cAPACITY = 100000, List<string> gROUPTRANSFERREQUIREMENTS = default, int mAXINVITESREQUESTS = 50, int mAXJOINED = 100, int mAXJOINEDPLUS = 200, int mAXLANGUAGES = 10, int mAXLINKS = 3, int mAXMANAGEMENTROLES = 5, int mAXOWNED = 5, int mAXROLES = 50)
```

Comment:
the variable casing is inverted, i guess someone had capslock on

Reply:
This is because in the spec, these are all capitlized and the generator tries to camelCase them automatically.